grakn-graph/src/main/java/ai/grakn/factory/SystemKeyspace.java

```diff
@@ -186,8 +198,8 @@ public static void deleteKeyspace(String keyspace){
      * only consists of types, the inserts are idempotent and it is safe to load it
      * multiple times.
      */
-    private static void loadSystemOntology() {
-        try (GraknGraph graph = factory().open(GraknTxType.WRITE)) {
+    private static void loadSystemOntology(InternalFactory factory) {
```

Comment:
Why pass in the factory now? This would essentially allow you to have multiple factories (graph connections) servicing the system keyspace. This is not ideal. The factory should be set internally to ensure only 1 graph connection can be made to the system keyspace.